_data/simple-icons.json

```diff
@@ -13644,6 +13644,11 @@
             "source": "https://design.ubuntu.com/resources",
             "guidelines": "https://design.ubuntu.com/brand"
         },
+        {
+            "title": "Ubuntu MATE",
+            "hex": "84A454",
+            "source": "https://github.com/ubuntu-mate/ubuntu-mate-artwork/blob/7920d3f2c810108e3032a3dc31335d145d785645/usr/share/icons/Yaru-MATE-light/256x256%402x/places/start-here.png"
```

Comment:
There is an SVG available on the website homepage. We should probably use that as the source here.

Reply:
This is outdated, because I've changed it to the correct URL.